test-fixtures/update-rust-component-version.py

```diff
@@ -42,6 +44,45 @@ def read_rust_components_tag_version(SPM_PACKAGE):
         logging.info(f"Unexpected error: {e}")
     return None, None
 
+def focus_read_rust_components_tag_version():
```

Comment:
I'm wondering, instead of implementing `focus_read_rust_components_tag_version()` separately, can you reuse `read_rust_components_tag_version()`, and pass it the Focus SPM package file?

Something like this:

```python
focus_current_tag, focus_current_commit = read_rust_components_tag_version(FOCUS_SPM_PACKAGE)
```

Reply:
Oh, gotcha, it looks like Focus still uses version 1 of the `Package.resolved` format, but iOS uses version 2 now—thanks!